src/main/java/com/senkiv/bookstore/mapper/BookMapper.java

```diff
@@ -14,4 +17,19 @@ public interface BookMapper {
     Book update(CreateBookRequestDto dto, @MappingTarget Book book);
 
     BookDto toDto(Book model);
+
+    default Map<String, String> toMap(BookSearchParametersDto dto) {
```

Comment:
It’s not a flexible solution. Think about how you would extend it if you needed to add a price filter? It's better to follow the solution proposed in the video tutorial.